deprecate: Deprecate ip_addresses attribute in project resource and data source (#2541)
mongodb/terraform-provider-mongodbatlas: docs/data-sources/project.md

* `tags` - Map that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the project. To learn more, see [Resource Tags](https://www.mongodb.com/docs/atlas/tags/)
* `teams` - Returns all teams to which the authenticated user has access in the project. See [Teams](#teams).
* `limits` - The limits for the specified project. See [Limits](#limits).

> * `ip_addresses` - IP addresses in a project categorized by services. See [IP Addresses](#ip-addresses). **WARNING:** this attribute is deprecated and will be removed in version 1.21.0. Please transition to using `mongodbatlas_project_ip_addresses` data source.

Reviewer: ```suggestion
* `ip_addresses` - IP addresses in a project categorized by services. See [IP Addresses](#ip-addresses). **WARNING:** this attribute is deprecated and will be removed in version 1.21.0. Use the `mongodbatlas_project_ip_addresses` data source instead.
```

Reviewer 2: same for this suggestion.
